elastic/kibana
# [Infrastructure UI] Processes tab: add N/A if the process summary is missing

Closes [#153942](https://github.com/elastic/kibana/issues/153942)

## Summary

This PR shows 'N/A' when the values inside the process summary are not available (and the actual value when available)


## Testing
- Open Hosts view and find a host ( where the process summary is not available ) 
- Open the flyout and select the "Processes" tab
- The values inside the process summary should be 'N/A'
(Same should be shown in inventory)

<img width="2259" alt="Screenshot 2023-05-11 at 13 34 58" src="https://github.com/elastic/kibana/assets/14139027/7f8a6e3d-cd87-452a-9f73-13dbc567501d">


<img width="2258" alt="Screenshot 2023-05-11 at 13 36 05" src="https://github.com/elastic/kibana/assets/14139027/2708ef2f-b119-46a4-80e8-048e25a22c21">

- Open Hosts view and find a host ( where the process summary is available ) 
- Open the flyout and select the "Processes" tab
- The values inside the process summary should be available (and total processes > 0)
(Same should be shown in inventory)

<img width="2262" alt="Screenshot 2023-05-11 at 13 35 33" src="https://github.com/elastic/kibana/assets/14139027/89aeb53f-c7c2-483f-b21f-9a0b67ca22c4">
 
<img width="2265" alt="Screenshot 2023-05-11 at 13 36 29" src="https://github.com/elastic/kibana/assets/14139027/06b44138-21bb-4c47-a31b-b43749df58a0">
